CHANGES.md

```diff
@@ -12,6 +12,9 @@
 
 ### Security
 
+- Use error types instead of relying on exceptions in buffer operations
```

Comment:
I don't think it should be in `Security` as I would reserve this for serious issues such as leaking passwords or cryptographic keys.

If these errors could leak to users I'd say it should be in `Fixed` with a message that speaks to users, something like:
`Fix uncaught exceptions in distrib subcommand and replace them with proper error messages`.

Otherwise it probably shouldn't even be in the changelog!

Reply:
done